addons/payment_razorpay/controllers/razorpay_auth.py

```diff
@@ -0,0 +1,100 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+
+from odoo import _, exceptions, fields, http
+from odoo.addons.iap.tools import iap_tools
+from odoo.http import request
+from datetime import timedelta
+
+from werkzeug.exceptions import Forbidden
+from werkzeug.urls import url_join
+from werkzeug.utils import redirect
```

Comment:
```suggestion
import logging

from odoo import _, exceptions, fields, http
from odoo.addons.iap.tools import iap_tools
from odoo.http import request
from datetime import timedelta

from werkzeug.exceptions import Forbidden
from werkzeug.urls import url_join
from werkzeug.utils import redirect
```

https://www.odoo.com/documentation/17.0/contributing/development/coding_guidelines.html?highlight=coding%20guid#imports